pkg/acceptance/bettertestspoc/assert/objectparametersassert/user_parameters_snowflake_gen.go

```diff
@@ -0,0 +1,1251 @@
+	return u.HasDefaultParameterValue(sdk.UserParameterTimestampInputFormat)
+}
+
+func (u *UserParametersAssert) HasDefaultTimestampLtzOutputFormatValue() *UserParametersAssert {
+	return u.HasDefaultParameterValue(sdk.UserParameterTimestampLtzOutputFormat)
+}
+
+func (u *UserParametersAssert) HasDefaultTimestampNtzOutputFormatValue() *UserParametersAssert {
+	return u.HasDefaultParameterValue(sdk.UserParameterTimestampNtzOutputFormat)
+}
+
+func (u *UserParametersAssert) HasDefaultTimestampOutputFormatValue() *UserParametersAssert {
+	return u.HasDefaultParameterValue(sdk.UserParameterTimestampOutputFormat)
+}
+
+func (u *UserParametersAssert) HasDefaultTimestampTypeMappingValue() *UserParametersAssert {
+	return u.HasDefaultParameterValue(sdk.UserParameterTimestampTypeMapping)
+}
+
+func (u *UserParametersAssert) HasDefaultTimestampTzOutputFormatValue() *UserParametersAssert {
+	return u.HasDefaultParameterValue(sdk.UserParameterTimestampTzOutputFormat)
+}
+
+func (u *UserParametersAssert) HasDefaultTimezoneValue() *UserParametersAssert {
+	return u.HasDefaultParameterValue(sdk.UserParameterTimezone)
+}
+
+func (u *UserParametersAssert) HasDefaultTimeInputFormatValue() *UserParametersAssert {
+	return u.HasDefaultParameterValue(sdk.UserParameterTimeInputFormat)
+}
+
+func (u *UserParametersAssert) HasDefaultTimeOutputFormatValue() *UserParametersAssert {
+	return u.HasDefaultParameterValue(sdk.UserParameterTimeOutputFormat)
+}
+
+func (u *UserParametersAssert) HasDefaultTraceLevelValue() *UserParametersAssert {
+	return u.HasDefaultParameterValue(sdk.UserParameterTraceLevel)
+}
+
+func (u *UserParametersAssert) HasDefaultTransactionAbortOnErrorValue() *UserParametersAssert {
+	return u.HasDefaultParameterValue(sdk.UserParameterTransactionAbortOnError)
+}
+
+func (u *UserParametersAssert) HasDefaultTransactionDefaultIsolationLevelValue() *UserParametersAssert {
+	return u.HasDefaultParameterValue(sdk.UserParameterTransactionDefaultIsolationLevel)
+}
+
+func (u *UserParametersAssert) HasDefaultTwoDigitCenturyStartValue() *UserParametersAssert {
+	return u.HasDefaultParameterValue(sdk.UserParameterTwoDigitCenturyStart)
+}
+
+func (u *UserParametersAssert) HasDefaultUnsupportedDdlActionValue() *UserParametersAssert {
+	return u.HasDefaultParameterValue(sdk.UserParameterUnsupportedDdlAction)
+}
+
+func (u *UserParametersAssert) HasDefaultUseCachedResultValue() *UserParametersAssert {
+	return u.HasDefaultParameterValue(sdk.UserParameterUseCachedResult)
+}
+
+func (u *UserParametersAssert) HasDefaultWeekOfYearPolicyValue() *UserParametersAssert {
+	return u.HasDefaultParameterValue(sdk.UserParameterWeekOfYearPolicy)
+}
+
+func (u *UserParametersAssert) HasDefaultWeekStartValue() *UserParametersAssert {
+	return u.HasDefaultParameterValue(sdk.UserParameterWeekStart)
+}
+
+/////////////////////////////////////////////
+// Parameter explicit default value checks //
```

Comment:
I am not sure if we need them. I can see some use cases for them: e.g. instead of checking if the value is the same as the default value from the parameter response, we may want to assert certain value in some of the tests. We can remove them, or we can just leave them and see (both options are zero additional work in the current state)

Reply:
Left them for now